build/configs/rtl8730e/loadable_apps_pm_dev/defconfig

```diff
@@ -23,9 +23,11 @@ CONFIG_FRAMEWORK_DIR="../framework"
 CONFIG_TOOLS_DIR="../tools"
 # CONFIG_BUILD_FLAT is not set
 CONFIG_APP_BINARY_SEPARATION=y
-CONFIG_NUM_APPS=2
+CONFIG_NUM_APPS=1
```

Comment:
do we need for pm defconfig?

Reply:
https://github.com/Samsung/TizenRT/pull/6185#discussion_r1645976419

Now, PM + SMP are stable together, it's better to drop these ```loadable_apps_pm_dev``` defconfig.

```loadable_apps``` -   Will have smp, xip and single app, common binary and PM.

could you please let me know your opinion on these?

Reply:
Hello @sunghan-chang @kishore-sn , Changes we updated in this PR.

-  ```loadable_apps``` : It will have smp, pm, and xip with required changes (common bin and single app).
-  ``` loadable_apps_xip``` and ```loadable_apps_smp``` : dropped these defconfigs as we have added it's feature(xip and smp) to  ```loadable_apps```
-  No changes to any other defconfigs.

Let me know if we need any more changes.

Reply:
Hi @sunghan-chang 
In my opinion, this seems to be fine. Can you pls merge it?

Reply:
@neel-samsung @kishore-sn @abhishek-samsung The loadable_ext should enable loadable xip as well. libcxx exception is the same. Let's enable it. Let's keep the same configuration for internal repo's config for bixby+lcd as possible.

Reply:
I have added xip, smp and libcxx exception configs to loadable_ext as well. Please check and review.